node/service/src/relay_chain_selection.rs

```diff
@@ -179,34 +171,37 @@ where
 		maybe_max_number: Option<BlockNumber>,
 	) -> Result<Option<Hash>, ConsensusError> {
 		let longest_chain_best =
-			self.fallback.finality_target(target_hash, maybe_max_number).await?;
+			self.longest_chain.finality_target(target_hash, maybe_max_number).await?;
 
-		if self.selection.overseer.is_disconnected() {
+		if self.is_relay_chain {
```

Comment:
Shouldn't this be `!self.is_relay_chain`?

Reply:
You are absolutely correct. Ty!